vxquery-core/pom.xml

```diff
@@ -247,7 +247,7 @@
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
-        </dependency>
+            </dependency>
```

Comment:
code style... You can change the xml style setting to format this page automatically. Hyracks has a mailing list post about xml style we have been using.